src/collections/members/aviral-asthana/index.mdx

```diff
@@ -0,0 +1,15 @@
+---
+name: Aviral Asthana
+position: Contributor
+image_path: ./Aviral-Asthana.webp
+github: Aviral0702
+linkedin: aviral-asthana-02b70824b
+layer5: 3f770688-e487-45c2-943b-c94b950d3d69
+location: Ghaziabad, India
+bio: A passionate Web Developer and Competitive Programmer from India
```

Comment:
Aviral Asthana is a Computer Science student with a passion for full-stack web development. Proficient in the MERN stack, Aviral has hands-on experience in building robust applications and is keenly interested in Golang, Docker, and Kubernetes. He is dedicated to continuous improvement through contributions to open-source projects and actively engaging with the developer community.

Can you please update the bio with this one?

Reply:
Updated. 👍